cos-fleetshard-operator-debezium/src/test/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandControllerTest.java

```diff
@@ -59,6 +60,11 @@
     private static final String SCHEMA_REGISTRY_ID = "9bsv0s0k8lng031se9q0";
 
     private static final DebeziumOperandConfiguration CONFIGURATION = new DebeziumOperandConfiguration() {
+        @Override
+        public LocalObjectReference imagePullSecretsName() {
+            return null;
```

Comment:
Could you add another tests that comes with an actual value and 2 integration tests that test the default value and with an set value in the application.properties?